config/hackathon.yml

```diff
@@ -1,9 +1,17 @@
 defaults: &defaults
+  # Logic config
   registration_is_open: false
   event_is_over: true
   last_day_to_apply: <%= Date.new(2017, 2, 11) %>
   auto_late_waitlist: true
 
+  # Content config
+  name: BrickHack
+  logo_asset: brickhack.png
+  agreement_pdf_asset: BrickHack_ReleaseAgreement.pdf
+  email_from: '"BrickHack" <hello@codeRIT.org>'
```

Comment:
Still needs to be updated (#538), but better to do in one fell swoop in a single PR than only partially here